dstat.yaml

```diff
@@ -0,0 +1,46 @@
+package:
+  name: dstat
+  version: 0.7.4
+  epoch: 0
+  description: A versatile resource statistics tool
+  copyright:
+    - license: GPL
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - python3-dev
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/dagwieers/dstat/archive/v${{package.version}}.tar.gz
+      expected-sha512: d100223887ebd83d0fd1259436f39419a85df9002556abcfc8e4195aa63be73d85707dcd5c4fb56a979b29131485dce6e97c177fbfca08dc50bd0f92b15cc6f5
+
+  - uses: patch
+    with:
+      # The patch source is https://github.com/efexgee/dstat/commit/220a785321b13b6df92a536080aca6ef1cb644ad.patch
+      patches: 220a785321b13b6df92a536080aca6ef1cb644ad.patch
```

Comment:
Why are we including this patch?

Reply:
I used Arch Linux PKGBUILD to build this: _https://gitlab.archlinux.org/archlinux/packaging/packages/dstat/-/blob/main/PKGBUILD_

Reply:
Right — but that doesn't explain what the purpose of the patch is. We need to make that clear here in this file, or we should leave the patch out. Friendly reminder, Wolfi's reason for including a patch should not be "because another distro did it". 😃 

At face value, I see taking a tag from 2019 and adding in a patch from 2001, and it doesn't make sense. If we have a story to make it make sense, that'd be great!

Reply:
it says _"Subject: [PATCH] Fixed bug when specifying a delay"_ in the patch file, is that enough information for us to keep it?

Reply:
I think the bug seems not fixed in the upstream, so I assume that we may need that patch still.

_https://github.com/dstat-real/dstat/blob/e428c7dc7137f81f8ca6bef8854b37a4ddd4f337/dstat#L2700_